python/tvm/topi/cumsum.py

```diff
@@ -84,12 +93,18 @@ def gen_ir(data_buf, out_buf):
             i = fused // axis_mul_after
             j = fused % axis_mul_after
             base_idx = i * cumsum_axis_len * axis_mul_after + j
-            out_buf[base_idx] = maybe_cast(data_buf[base_idx])
+            if exclusive == 0:
+                out_buf[base_idx] = maybe_cast(data_buf[base_idx])
+            else:
+                out_buf[base_idx] = 0.0
```

Comment:
This will likely give an error if the input is an integer tensor. Need to cast 0 to `dtype`